Cargo.lock

```diff
@@ -49,9 +49,9 @@ dependencies = [
 
 [[package]]
 name = "aho-corasick"
-version = "1.0.4"
+version = "1.0.5"
```

Comment:
Seems you have accidentally bumped all dependencies in this PR. Would you please revert these changes?